### PR TITLE
Fixed Probe Linebreaks Across All Modules

### PR DESCRIPTION
**Pull request by huzaifakhan04**

### CODE REVIEW

File: recipes/journey_1/module_2/probe.yaml
Comment (mikhailocampo) [inline recipes/journey_1/module_2/probe.yaml:1]: Please apply the node fixes described. Replace all of the "END" nodes that currently are the "example" screens based on the wording.

You can use the outlined message as the content for the END node. Ensure that the example screen has a single CTA which points to the new END node accordingly.
image

Comment (mikhailocampo) [inline recipes/journey_1/module_2/probe.yaml:1]: We should utilize the double newline as described. Here is a valid one:
image

And the one that I see in the current probe which only use one newline.
image

Comment (mikhailocampo) [reply recipes/journey_1/module_2/probe.yaml:1]: In short, add one more newline to each question

**Review by mikhailocampo — CHANGES_REQUESTED** [review]
Please see requested changes

**Comment by bkb-Git** [comment]
@mikhailocampo @hash2004 @huzaifakhan04 Please don't merge this PR yet, I'm checking if the formatting will work as expected.

**Comment by bkb-Git** [comment]
@mikhailocampo @hash2004 @huzaifakhan04 , works as expected, this is all good on my end. Tested Module 3 Probe
image

**Comment by mikhailocampo** [comment]
@bkb-Git @hash2004 When you guys have a format that you're satisfied with, I can approve the merge

**Pull request closed without merging**
